webapp/stores/emoji_store.jsx

```diff
@@ -148,8 +149,7 @@ class EmojiStore extends EventEmitter {
 
     getEmojiImageUrl(emoji) {
         if (emoji.id) {
-            // must match Client.getCustomEmojiImageUrl
-            return `/api/v3/emoji/${emoji.id}`;
+            return Client.getCustomEmojiImageUrl(emoji.id);
```

Comment:
Can you revert this or maybe update the comment? I had to duplicate this code because the unit tests fail when they import the `Client`.

I need to look into why that didn't cause Jenkins to fail the build for this though. That's kind of worrying that they're not running any more.